api/v1beta1/flinkcluster_types.go

```diff
@@ -365,11 +365,14 @@ type CleanupPolicy struct {
 // JobSpec defines properties of a Flink job.
 type JobSpec struct {
 	// JAR file of the job.
-	JarFile string `json:"jarFile"`
+	JarFile *string `json:"jarFile,omitempty"`
 
 	// Fully qualified Java class name of the job.
 	ClassName *string `json:"className,omitempty"`
 
+	// Python file of the job.
+	Python *string `json:"python,omitempty"`
```

Comment:
wdyt of `PythonFile` inline with `JarFile`

Reply:
It seems good to apply your suggestion